lib/bundlex/project.ex

```diff
@@ -40,6 +40,7 @@ defmodule Bundlex.Project do
           includes: [String.t()],
           lib_dirs: [String.t()],
           libs: [String.t()],
+          os_deps: [Bundlex.Native.os_dep()],
```

Comment:
It's not documented in the typedoc. Right now, this typedoc is copied to readme so we have to update both places, maybe we can replace it with an example and a link to the typedoc in the readme?